docs/source/reference/numpysupported.rst

```diff
@@ -182,6 +182,7 @@ floating-point and complex numbers:
 * :func:`numpy.linalg.eigvalsh` (only the first argument).
 * :func:`numpy.linalg.inv`
 * :func:`numpy.linalg.lstsq`
+* :func:`numpy.linalg.matrix_power`
```

Comment:
hmmm, perhaps we ought to add a note encompassing all `np.linalg` to the same effect? All these LAPACK backed functions only support `[s,d,c,z]` cf. the underlying impls.